CatAnalyzer/python/leptonSF_cff.py

```diff
@@ -10,40 +10,73 @@
 
 ## Muon SF reference https://twiki.cern.ch/twiki/bin/view/CMS/MuonWorkInProgressAndPagResults
 ## SF for Run2016BCDE, before HIP issue
-muonSFTight = cms.PSet(
-    # Values of "MC_NUM_TightID_DEN_genTracks_PAR_pt_eta + TightISO_TightID_pt_eta"
-    pt_bins = cms.vdouble(20, 25, 30, 40, 50, 60, 120),
-    abseta_bins = cms.vdouble(0, 0.9, 1.2, 2.1, 2.4),
+muonSFTightIdOnly = cms.PSet(
```

Comment:
There are analyzer modules requires muonSFTight. I'll make a PR to make combined SF in the python level.